conjure/cycles/graph_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2018 Palantir Technologies. All rights reserved.
```

Comment:
Since this is a new file, year should be 2023 (the `./godelw verify` should automatically add the current year if the package has no comment/header, but I'm assuming this was copy-pasted)